### DISCUSSION TITLE
Measuring Cityhub current and voltage

### DISCUSSION THREAD

**Discussion by Dr-Paul57** [pybricks/support#1434] [Q&A]
In the documentation for the city hub it stated that both current and voltage for the hub can be measured.
How do I do that?

**Comment by dlech** [comment]
```python
...

v = hub.battery.voltage()
i = hub.battery.current()
print(f"{v}mv, {i}mA")
```

**Comment by Dr-Paul57** [comment]
Thank you.   I forgot (didn’t know) to put ‘hub’ before battery.voltage?current!
